src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -140,6 +146,30 @@ class FormPage extends React.Component {
       }
     }
 
+    // Bypass the SchemaForm and render the custom component
+    // NOTE: I don't think FormPage is rendered on the review page, so I believe
+    // onReviewPage will always be false here
+    if (typeof route.pageConfig.CustomPage === 'function') {
+      return (
+        <div className={pageClasses}>
+          <route.pageConfig.CustomPage
+            name={route.pageConfig.pageKey}
+            title={route.pageConfig.title}
+            data={data}
+            pagePerItemIndex={params ? params.index : undefined}
+            isLoggedIn={formContext?.isLoggedIn}
+            prefilled={formContext?.prefilled}
+            onReviewPage={formContext?.onReviewPage}
+            trackingPrefix={this.props.form.trackingPrefix}
+            uploadFile={this.props.uploadFile}
+            onChange={this.onChange}
+            goBack={this.goBack}
+            goForward={this.onSubmit}
```

Comment:
Where is this API coming from?

Reply:
My head. :stuck_out_tongue_closed_eyes: 

It's really whatever we decide it is. I put these in here because I thought they'd be the most useful. The way I came up with these is by copying the props to `SchemaForm` and going through the list, determining which ones would be useful. I renamed `onSubmit` and didn't include some of the others like `callOnContinue` which just served as a way to call some functions defined in the `formConfig`.